src/kafka-connect/home/home.controller.js

```diff
@@ -27,6 +27,8 @@ angularAPP.controller('HomeCtrl', function ($scope, $log, $http, env, constants,
 
 
                 angular.forEach(topics, function(t) {
+                     t = t + ' '
```

Comment:
Add a comment above this line:

// We are adding one space to the topic names when we add them to the diagram to avoid cyclic references when the topic name matches a connector name